src/quo2/components/wallet/account_card/style.cljs

```diff
@@ -0,0 +1,90 @@
+(ns quo2.components.wallet.account-card.style
+  (:require
+    [quo2.foundations.colors :as colors]))
+
+(defn text-color
+  [watch-only?]
+  (if (and watch-only? (not (colors/dark?)))
+    colors/neutral-100
+    colors/white))
+
+(defn card
+  [customization-color watch-only?]
+  {:width              161
+   :height             88
+   :background-color   (if watch-only?
+                         (colors/theme-colors colors/neutral-80-opa-5 colors/neutral-95)
+                         (colors/custom-color-by-theme customization-color 50 60))
+   :border-radius      16
+   :border-width       1
+   :border-color       (if watch-only?
+                         (colors/theme-colors colors/neutral-80-opa-5 colors/white-opa-5)
+                         (colors/custom-color-by-theme customization-color 50 60))
+   :padding-horizontal 12
+   :padding-top        8
+   :padding-bottom     10})
+
+(def profile-container
+  {:margin-bottom  8
+   :flex-direction :row})
+
+(def metrics-container
+  {:flex-direction :row
+   :align-items    :center})
+
+(def title
+  {:margin-bottom 9})
+
+(defn account-name
+  [watch-only?]
+  {:style  {:color       (text-color watch-only?)
+            :margin-left 5}
+   :size   :paragraph-2
```

Comment:
These are not style properties. Please move `size` and `weight` outside this namespace. Vars `metrics`, `account-value` and `add-account-title` also should be changed.

Reply:
Thanks!